modules/apps/translation/translation-web/src/main/resources/META-INF/resources/import_translation.jsp

```diff
@@ -23,89 +23,45 @@ portletDisplay.setShowBackIcon(true);
 portletDisplay.setURLBack(importTranslationDisplayContext.getRedirect());
 
 renderResponse.setTitle(LanguageUtil.get(resourceBundle, "import-translation"));
+
+String exceptionErrorMessage = null;
```

Comment:
I don't like this usage of the `exceptionErrorMessage` variable. I think it would be better to get remove this error logic from the JSP (`error` tag usage + variable), and create a method in the display context that returns the expected value. Something like:
```
public String getErrorMessage() {
    if (!MultiSessionErrors.contains(_portletRequest, XLIFFFileException.MustBeValid.class.getName())) {
        return null;
    }

    return LanguageUtil.get(_request, "please-enter-a-file-with-a-valid-xliff-file-extension");
}
```
and call this method when creating the data context for the react component.

Reply:
thanks @adolfopa I can try

Reply:
done! ba794dc